intl-config.js

```diff
@@ -1,5 +1,7 @@
-{
-  "supportedLocales": ["en-US", "de", "fr", "pt-BR", "es", "id"],
+require('habitat').load();
+
+module.exports = {
+  "supportedLocales": process.env.SUPPORTED_LOCALES || ["en-US", "de", "fr", "pt-BR", "es", "id"],
```

Comment:
I don't want to default this to `*` since our production is not going to have this enabled by default.